drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-kogito-client/src/main/java/org/drools/workbench/screens/scenariosimulation/kogito/client/editor/ScenarioSimulationEditorKogitoWrapper.java

```diff
@@ -133,6 +139,22 @@ public ScenarioSimulationEditorKogitoWrapper(
         this.scenarioSimulationKogitoDMNMarshallerService = scenarioSimulationKogitoDMNMarshallerService;
     }
 
+    @PostConstruct
+    public void init() {
+        scenarioSimulationEditorPresenter.getView().onResize();
+        HTMLCollection<Element> elements = DomGlobal.document.getElementsByClassName("tab-content");
+        if(elements.length == 1) {
+            HTMLDivElement element = (HTMLDivElement) elements.getAt(0);
+            element.style.height = CSSProperties.HeightUnionType.of("calc(100vh - 36px)");
+        }
+
+        DomGlobal.document.body.onresize = p0 -> {
```

Comment:
@treblereel Is it necessary to put that handler logic on `body` tag? I tried to put it on the `element` you defined and the fix seems working as well. I think we should avoid putting that handler in `body`, if possible.